src/components/a-form-input/_preview.html

```diff
@@ -0,0 +1,7 @@
+<axa-form-input id="test" value="Pre-filled content" name="test-name"></axa-form-input>
+<axa-form-input id="test" value="Pre-filled content" name="test-name" inline></axa-form-input>
+<axa-form-input id="test" value="Pre-filled content" name="test-name" disabled></axa-form-input>
```

Comment:
also for webcomponents id cannot be duplicated on a page. I would suggest, to not allow id to be set from outside. If we need one later, we can generate on the go a uuid. in `component-types` there is a static function for uuid generation that can be used (`BaseComponent.uuidv4()`).

here especially in the input-form we wont need an id as you packed wisely the input field inside its label